### ISSUE TITLE
account.billing_info raises AttributeError

### ISSUE THREAD

**Issue by vincenttheeten** [recurly/recurly-client-python#98]
Account.billing_info is never filled in
(using version 2.2.6)

```
import recurly
import time

recurlyApiKey = 'FILLIN'
recurlyPrivateKey = 'FILLIN'
recurly.API_KEY = recurlyApiKey
recurly.js.PRIVATE_KEY = recurlyPrivateKey

accountCode = "test-%s" % time.clock()

account = recurly.Account(account_code=accountCode)
account.email = 'verena@example.com'
account.first_name = 'Verena'
account.last_name = 'Example'
account.save()

account = recurly.Account.get(accountCode)
billing_info = account.billing_info  # fails with AttributeError
billing_info.token_id = 'TOKEN_ID'
billing_info.save()
```

**Comment by pdumoulin** [comment]
I am a developer at [JWPlayer](http://jwplayer.com), this bug is affecting some of our current customers and blocks us from expanding our Recurly usage for new customers.  Is there a work around or planned release for a fix?

**Comment by chrissrogers** [comment]
This is being reported as resolved in the latest version, `2.2.11`

**Comment by carlosescri** [comment]
It seems to be happening again in `2.2.12`... I was trying the same code used in the docs:

<https://dev.recurly.com/docs/create-an-accounts-billing-info-token>
